functions.php

```diff
@@ -17,8 +17,24 @@ function readYaml($file) {
     return $ret;
 }
 
+function getActions($dimensions) {
+    foreach ($dimensions as $dimension => $subdimensions) {
+        foreach ($subdimensions as $subdimension => $element) {
+            if ($subdimension == "_meta")
```

Comment:
This function get data from dimensions and skips _meta. This can be further customized to ignore other metadata sections (eg. in $element ...)